aem65/src/test/resources/component-configs/core-components/image/asset.yaml

```diff
@@ -0,0 +1,3 @@
+Asset:
+- type: IMAGE
+  value: majestic-rainbow.jpg
```

Comment:
Is this asset available in OOTB AEM :)?

Reply:
AEM or core-components-examples, not sure